src/control/xml/XmlPointNode.cpp

```diff
@@ -33,6 +55,24 @@ void XmlPointNode::writeOut(OutputStream* out)
 {
 	XOJ_CHECK_TYPE(XmlPointNode);
 
+	/**
+	 * Write timestamp before each stroke.
+	 * This way, when reading, we only need to store 
+	 * 1 timestamp value at a time 
+	 * and assign it to the consequent stroke.
+	 * By adding it this way we don't break 
+	 * xournal's fileformat backcompatibility
+	 */
+	out->write("<timestamp ");
```

Comment:
May there should be an option to completely disable timestamps

Reply:
Adding a check to write/load them only if a recording has been taken